ignore.go

```diff
@@ -2,7 +2,7 @@ package main
 
 import (
 	"bufio"
-	"fmt"
+	"log"
```

Comment:
`log.Print*` logs to `stderr` by default whereas the `fmt.Print*` commands log to `stdout`.
This change is required so the output of `godu` can be piped to e.g. `xargs`:

```sh
# This will try to remove a file named 'godu will walk through..' if we log to stdout
$ godu . | xargs rm
```